include/zephyr/drivers/flash/ra_flash_api_extensions.h

```diff
@@ -13,6 +13,11 @@ enum ra_ex_ops {
 	FLASH_RA_EX_OP_WRITE_PROTECT = FLASH_EX_OP_VENDOR_BASE,
 };
 
+enum qspi_flash_ex_op_types {
+	QSPI_FLASH_EX_OP_RESET = 0,
```

Comment:
There is already system wide id for reset
`FLASH_EX_OP_RESET = 0` https://github.com/zephyrproject-rtos/zephyr/blob/03959a20f7834ffe50a51df2b2e45ef87d1b610b/include/zephyr/drivers/flash.h#L640
and if the meaning is the same, the system wide one should be used.

Reply:
We fixed it as your advice.